<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>136</width>
            <height>215</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>VOLTAJE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>POSICION X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>POSICION Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>RED</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>GREEN </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>BLUE</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QSpinBox" name="id">
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="volt">
               <property name="maximum">
                <double>999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="pos_x">
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="pos_y">
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="red">
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="green">
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="blue">
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="busId">
          <property name="geometry">
           <rect>
            <x>137</x>
            <y>365</y>
            <width>243</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>BUSCAR POR ID</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>221</y>
            <width>130</width>
            <height>86</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="agregarFinal">
             <property name="text">
              <string>AGREGAR AL FINAL </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="agregarInicio">
             <property name="text">
              <string>AGREGAR AL INICIO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bmostrar">
             <property name="text">
              <string>MOSTRAR</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLineEdit" name="lineId">
          <property name="geometry">
           <rect>
            <x>386</x>
            <y>365</y>
            <width>244</width>
            <height>24</height>
           </rect>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="seeBox">
          <property name="geometry">
           <rect>
            <x>137</x>
            <y>-1</y>
            <width>243</width>
            <height>309</height>
           </rect>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>386</x>
            <y>-1</y>
            <width>244</width>
            <height>309</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="or_Id">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>330</y>
            <width>111</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>ORDENAR POR ID</string>
          </property>
         </widget>
         <widget class="QPushButton" name="or_vol">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>330</y>
            <width>131</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>ORDENAR POR VOLTAJE</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>51</x>
            <y>14</y>
            <width>511</width>
            <height>341</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QGraphicsView" name="graphicsView"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>LIMPIAR</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>DIBUJAR</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="neuronaCercana">
             <property name="text">
              <string>CALCULAR NEURONA M√ÅS CERCANA</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionCargar"/>
    <addaction name="actionguardar"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar</string>
   </property>
  </action>
  <action name="actionguardar">
   <property name="text">
    <string>guardar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
